.github/workflows/rector.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
             -
                 # commit only to core contributors who have repository access
                 if: github.event.pull_request.head.repo.full_name == github.repository
-                uses: EndBug/add-and-commit@v8.0.1
+                uses: EndBug/add-and-commit@v7.5.0
```

Comment:
Is this on purpose?

Reply:
I see https://github.com/rectorphp/rector-src/pull/1756#issuecomment-1027597913 :)